pangres/engine.py

```diff
@@ -127,7 +127,7 @@ def __init__(self,
         # all index levels have names
         index_names = list(df.index.names)
         if any(ix_name is None for ix_name in index_names):
-            raise UnnamedIndexLevelsException("All index levels must be named!")
+            raise UnnamedIndexLevelsException("All index levels must be named! Ensure you used `df = df.set_index(['col1', 'col2', ...])` before.")
```

Comment:
Hey, that looks good but the line is too long (must be <= 120 characters), so I suggest doing so:

```suggestion
        raise UnnamedIndexLevelsException("All index levels must be named! "
                                          "Ensure you used `df = df.set_index(['col1', 'col2', ...])` before.")
```